modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -203,6 +218,21 @@ definition {
 		return ${response};
 	}
 
+	macro putObjectEntryById {
```

Comment:
Same here, please check if this is already exists in ObjectDefinitionAPI.macro, I think it's better to move it to the CustomAPI.macro or just using ObjectDefinitionAPI if the method exist, thanks!

Reply:
this method doesn't exist in ObjectDefinitionAPI.macro